### DISCUSSION TITLE
Warning Message: Addon 'acre_sys_sog' requires addon 'data_f_vietnam'

### DISCUSSION THREAD

**Discussion by CYBER-MISFIT** [IDI-Systems/acre2#1100] [Q&A] [answered]
Getting this error and dont know how to solve it, im using serverblend as a hosting site.

**Comment by jonpas** [comment, marked as answer]
Load `@acre2/optionals/@acre_sys_sog`.

**Reply by CYBER-MISFIT** [reply]
So it loads the Acre sys sog fine, but its trying to call on data_f_vietnam, and cant find it

**Reply by PabstMirror** [reply]
- if you are trying to play with new cdlc
make sure it's downloaded
add `vn;` to mod list
- if you aren't
then delete the acre compat for it
